### PR TITLE
Replaced concats with template literals

### PR DESCRIPTION
**Pull request by kajes**
Fixes #

Changes proposed in this pull request:
- 
- 

Checklist:
- [ ] docs updated
- [ ] tests updated
- [ ] [Changes](https://github.com/haraka/Haraka/blob/master/Changes.md) updated

### CODE REVIEW

File: connection.js [hunk 1]
```diff
@@ -296,24 +296,24 @@ Connection.prototype.process_line = function (line) {
     if (this.state >= states.DISCONNECTING) {
         if (logger.would_log(logger.LOGPROTOCOL)) {
             this.logprotocol(
-                "C: (after-disconnect): " +
-                    this.current_line, {'state': this.state});
+                `C: (after-disconnect):
```
Comment (msimerson) [inline connection.js]: this introduces a newline character

File: connection.js [hunk 2]
```diff
@@ -678,7 +679,7 @@ Connection.prototype.init_transaction = function (cb) {
         self.transaction = trans.createTransaction(self.tran_uuid());
         // Catch any errors from the message_stream
         self.transaction.message_stream.on('error', function (err) {
-            self.logcrit('message_stream error: ' + err.message);
+            self.logcrit(`message stream error: ${error.message}`);
```
Comment (msimerson) [inline connection.js]: mismatched variable name

(look at the travis test failures)

**Comment by kajes** [comment]
Right, so those should be fixed now.

**Comment by msimerson** [comment]
And they are, but you've got a lint error. Follow the 'Details' link to the right of Travis-CI and it'll lead you to the [test run](https://travis-ci.org/haraka/Haraka/jobs/282071064) which shows you the lint error.

**Comment by msimerson** [comment]
Sorry, this has collided with another PR that has already landed. Best to pick another file to work in.

**Pull request closed without merging**